axis_linear:
  x: 1
  y: 0
  depth_up: 5
  depth_down: 2
scale_linear:
  x: 0.2
  y: -0.2
  depth_up: -0.2
  depth_down: -0.2
offset_linear:
  x: 0
  y: 0
  depth_up: -1
  depth_down: -1
axis_angular:
  yaw: 3
scale_angular:
  yaw: -0.12
offset_angular:
  yaw: 0
arming_button: 9
attitude_control_button: 0
depth_hold_button: 1